core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -348,6 +352,8 @@ class MapUnit {
             if(currentTurnsFortified<2)
                 action = action!!.replace(currentTurnsFortified.toString(),(currentTurnsFortified+1).toString(), true)
         }
+        if (hasUnique("Heal adjacent units for an additional 2 HP per turn"))
+            currentTile.neighbors.flatMap{ it.getUnits() }.forEach{ it.healBy(2) }
```

Comment:
There's no way this is just 2 health, maybe 20? Also, I'm not sure this heals all units, maybe it's like medic and units that are already talking get healed more, worth asking on discord

Reply:
The [Fandom wiki](https://civilization.fandom.com/wiki/Khan_(Civ5)) reports 2 HP in vanilla (updated to 15 in BNW e G&K)